cmd/identities/validate.go

```diff
@@ -0,0 +1,138 @@
+package identities
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/markbates/pkger"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tidwall/gjson"
+
+	"github.com/ory/kratos/internal/clihelpers"
+
+	"github.com/ory/jsonschema/v3"
+	"github.com/ory/kratos/cmd/cliclient"
+	"github.com/ory/kratos/internal/httpclient/client"
+	"github.com/ory/kratos/internal/httpclient/client/common"
+	"github.com/ory/x/viperx"
+)
+
+var validateCmd = &cobra.Command{
+	Use:   "validate <file.json [file-2.json [file-3.json] ...]>",
+	Short: "Validate local identity files",
```

Comment:
Similarly to import, this should also support arrays. Please also add a description to make it a bit easier to understand what this command does. Maybe with an example output!